src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -3363,15 +3363,19 @@ public void foo()
 
 		//See: https://developer.android.com/about/versions/marshmallow/android-6.0-changes#behavior-apache-http-client
 		[Test]
+		[Retry (5)]
 		public void MissingOrgApacheHttpClient ([Values ("dx", "d8")] string dexTool)
 		{
+			string downloaddir = Path.Combine ("temp", TestName, "Downloads");
+			Directory.CreateDirectory (downloaddir);
```

Comment:
You might need to move this down and use `b.ProjectDirectory`:
```csharp
Path.Combine (Root, b.ProjectDirectory, "Downloads");
```
I'm not sure what the current directory would be here.